bundles/org.eclipse.ui.workbench/.settings/.api_filters

```diff
@@ -4,7 +4,6 @@
         <filter comment="https://github.com/eclipse-platform/eclipse.platform.ui/issues/1155" id="923795461">
             <message_arguments>
                 <message_argument value="3.131.0"/>
-                <message_argument value="3.130.0"/>
```

Comment:
I still don't get why this change is necessary. What happens without it?

Reply:
Build fails with "The major version should be incremented in version 3.131.0, since API breakage occurred since version 3.130.0"
This happens due to MultiPageEditor removal in previous push to master.
https://github.com/eclipse-platform/eclipse.platform.ui/pull/1253#discussion_r1375279272

<img width="1224" alt="Screenshot 2023-11-07 at 14 10 05" src="https://github.com/eclipse-platform/eclipse.platform.ui/assets/650857/0b8a9c03-94cb-416a-bf49-324245f015d0">

Reply:
Can't you just keep the current version in MANIFEST.MF? If I'm not mistaken, there is no externally visible change (all API changes are in internal code) and version was already bumped. So keeping same versions, and rebasing on master should not trigger new errors?

Reply:
The version is untocuhed

Reply:
I've fetched the PR locally and reverted the .api_filters to previous version and see no error.
Can you please try to leave it unmodified? If build succeeds without modification there, then it means we've worried for nothing and we can move forward.

Reply:
I've force-pushed a revision without this change.

Reply:
I've revisited the change history and now see, that it is a remnant from the attempt to bump workbench minor or patch versions. The bump is removed and the change might no longer be necessary.